nexus/db-queries/src/db/datastore/trust_quorum.rs

```diff
@@ -108,6 +124,10 @@ impl DataStore {
 
         let values: Vec<(DbTypedUuid<RackKind>, i64)> =
             dsl::trust_quorum_configuration
+                .filter(dsl::state.ne_all(vec![
+                    DbTrustQuorumConfigurationState::Committed,
+                    DbTrustQuorumConfigurationState::Aborted,
```

Comment:
How likely is it we'll add new `DbTrustQuorumConfigurationState` variants after this work lands? I'm mildly worried some of these spots may be tricky to find if we need to update them all for new variants (but am not sure it's worth doing work to help with that, hence the question).